tools/data/matting/comp1k/extend_fg.py

```diff
@@ -112,16 +115,14 @@ def main():
         os.makedirs(p, exist_ok=True)
 
     fg_names = osp.join(dir_prefix, f'{fname_prefix}_fg_names.txt')
+    save_json_path = f'{fname_prefix}_list.json'
```

Comment:
Will this override the DIM filelist?

Reply:
Yes.

Reply:
Change the filename to `f'{fname_prefix}_list_fba.json'`